tests/format/js/conditional/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -369,41 +378,28 @@ const reactRouterExampleNonJSX =
 
 inJSXExpressionContainer.withLongConditionals.example = (
   <div>
-    {(
-      isACat() &&
-      (someReallyLongCondition ||
-        moreInThisLongCondition)
-    ) ?
-      someReallyLargeExpression
-        .toMakeMeowNoise()
-        .willCauseParens()
-    : (
-      someReallyLongCondition ||
-        moreInThisLongCondition
-    ) ?
+    {isACat() && (someReallyLongCondition || moreInThisLongCondition) ?
+      someReallyLargeExpression.toMakeMeowNoise().willCauseParens()
+    : someReallyLongCondition || moreInThisLongCondition ?
       bark()
-    : someReallyLargeExpression
-        .toMakeMeowNoise()
-        .willCauseParens()
-    }
+    : someReallyLargeExpression.toMakeMeowNoise().willCauseParens()}
   </div>
 );
 
 inJSXExpressionContainer.withLoops.orBooleans.example = (
   <div>
     {items ?
-      items.map((item) => (
+      items.map((item) =>
         item.display ?
           <Item item={item} attr="breaks ternary but not consequent" />
-        : <Blank />
-      ))
+        : <Blank />,
```

Comment:
It's weird to put a `,` here where there's only one argument to the function. This looks like a regression.

Reply:
I don't think that's related to this PR, this is the input (I copy/pasted output to input so they're the same for easier reading), that's an unrelated opportunity for improvement.